polylogue/services/attachment_text.py

```
@@ -0,0 +1,182 @@
+    return text, truncated
+
+
+def _extract_image_text(path: Path, *, max_chars: int) -> Tuple[str, bool]:
+    try:
+        from PIL import Image
+        import pytesseract
+    except ImportError as exc:  # pragma: no cover - dependency required when OCR requested
+        raise RuntimeError(
+            "OCR requested for attachments but Pillow/pytesseract are not installed. "
+            "Install with `pip install 'polylogue[ocr]'` or add pillow+pytesseract to your environment."
+        ) from exc
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)